### PR TITLE
feature/beornia

### PR DESCRIPTION
**Pull request by Azfaa**
- Added Conquest of the Southern Deserts JE
- Added Seizing the Collasuyan Range JE
- Added Conqueror of the Cone JE

### CODE REVIEW

File: common/journal_entries/dvg_the_cone.txt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+		     owns_entire_state_region = STATE_NEW_CORNWALL
+		     owns_entire_state_region = STATE_COLLASUYU
+		}
+    }
+    
+
+	complete = {
+			#DVG
+			owns_entire_state_region = STATE_LANDS_END
+		}
```
Comment (MadJamez) [inline common/journal_entries/dvg_the_cone.txt:26]: NIT: Can you fix the formatting? And duplicate new lines.
image

File: events/seizing_collasuyu.txt [hunk 2]
```diff
@@ -0,0 +1,63 @@
+﻿namespace = seizing_collasuyu
+
+# Seizing the southern ranges JE completion
+seizing_collasuyu.1 = {
```
Comment (MadJamez) [inline events/seizing_collasuyu.txt:4]: Can you make new custom events start with dvg_ in the file name? Makes it easier when we need to apply vanilla patches on the mod.

**Review by MadJamez — CHANGES_REQUESTED** [review]
Looks good, have a few small remarks.

File: events/seizing_collasuyu.txt (hunk 2 above)
Comment (Azfaa) [reply events/seizing_collasuyu.txt]: Oh I didn't know you could resolve it here so manually changed things, I think its fixed now?

File: common/journal_entries/dvg_the_cone.txt (hunk 1 above)
Comment (MadJamez) [reply common/journal_entries/dvg_the_cone.txt]: image
Need to move 1 tab back on owns+ and then its fine.

**Comment by MadJamez** [comment]
@Azfaa Any reason for not including AI strategies for these journals?

**Comment by Azfaa** [comment]
> 

Main reason was falling behind main but now also with the new patch, I thought it easier to use current Main for AI Strategies based on the 1.04 than trying to do it myself due to lack of experience.

**Comment by MadJamez** [comment]
We should also make the first journal locked behind a decision requiring nationalism, just like all the other ones. Unless there is a reason for not doing this?
The 2nd journal can then still be unlocked when the first one is completed.

**Review by MadJamez — APPROVED** [review]

**Comment by Azfaa** [comment]
> @Azfaa Any reason for not including AI strategies for these journals?

I would like you to check the AI Strategy files, other than that I am ready to merge with main :)

**Pull request merged**